README.md

```diff
@@ -207,25 +207,28 @@ once the application starts.
 
 ## Run the application <a name="development-run"></a>
 
-1. [Set up the GitHub App](#github-app)
-2. [Set up the application](#setup)
+1. [Set up the GitHub App](#github-app). 
+2. [Set up the command-bot application](#setup)
```

Comment:
Why is this needed? "The application" can only mean command-bot here because this is its README